es/sources_index.go

```diff
@@ -362,12 +362,23 @@ func (index *SourcesIndex) indexSource(mdbSource *mdbmodels.Source, parents []st
 			authors := authorsByLanguage[i18n.Language]
 			s := append(authors, pathNames...)
 			leaf := s[len(s)-1]
+			if mdbSource.TypeID > 2 && i18n.Description.Valid && i18n.Description.String != "" {
+				if i18n.Language == "he" {
+					leaf = fmt.Sprintf("%s %s", leaf, i18n.Description.String)
+				} else {
+					leaf = fmt.Sprintf("%s. %s", leaf, i18n.Description.String)
```

Comment:
Why this "." difference is so important?
Add comment on why, an example with or without the dot.

Reply:
TBD add comment

Reply:
this logic been removen